extensions/amp-addthis/0.1/amp-addthis.js

```diff
@@ -137,6 +139,9 @@ class AmpAddThis extends AMP.BaseElement {
     const pubId = this.element.getAttribute('data-pub-id') || '';
     const widgetId = this.element.getAttribute('data-widget-id') || '';
     const productCode = this.element.getAttribute('data-product-code') || '';
+    if (this.win.innerWidth >= 979){
```

Comment:
Please use `this.getViewport().getWidth()`

Reply:
Fixed

Reply:
@aghassemi  do you know why `The command "node build-system/pr-check/checks.js" exited with 1.` this happened on continuous-integration/travis-ci/pr